src/helm/benchmark/metrics/vision_language/image_metrics.py

```diff
@@ -182,10 +178,17 @@ def evaluate_generation(
                 [self.LPIPS_SIMILARITY, self.lpips_similarity, image, ref_image],
                 [self.FID_SIMILARITY, self.fid_similarity, image, ref_image],
                 [self.SSIM_SIMILARITY, self.compute_ssim, gray_image, gray_ref_image],
-                # Text
-                [self.EDIT_SIMILARITY, self.compute_edit_sim, text, ref_text],
             ]
 
+            if "text" in annotation:
+                # For some tasks, we have the reference text. For those, compute the edit similarity for some signal.
+                # Although having the reference text is not a requirement for a task to be included in Image2Structure.
+                text: str = annotation["text"]
+                ref_text: str = reference.output.text
+                metric_runs.append(
+                    [self.EDIT_SIMILARITY, self.compute_edit_sim, text, ref_text],
+                )
```

Comment:
Ok this is good as you are right we might not want text metrics for all scenarios. However we might want to not include this or other text metrics in the future. I think the correct way to do this would be to define each metric as a couple of a name and an annotation field name and then verify that for each metric, we have the necessary annotations. This seems a bit out of scope for this PR so we should probably just merge this and I will make a quick PR to address this.